py/sync_examples.py

```diff
@@ -112,9 +112,10 @@ def main():
 
     example_items = [dict(slug=e.slug) for e in examples]
     example_items.insert(0, dict(slug='index'))
+    example_items.insert(1, dict(slug='gallery'))
     write_file(os.path.join(website_dir, 'examples.js'), f'module.exports={json.dumps(example_items)}')
-    # write_file(os.path.join(example_md_dir, 'index.md'), make_toc(examples))
-    write_file(os.path.join(example_md_dir, 'index.md'), make_gallery(examples))
+    write_file(os.path.join(example_md_dir, 'index.md'), make_toc(examples))
```

Comment:
The `make_toc()` function needs to use `useBaseUrl()` like the `make_gallery_thumbnail()` function. See #413

Reply:
@lo5 This was fixed by just using the relative path as `e.slug` instead of `examples/e.slug`. I think this should work without `useBaseUrl()` function. I tested this on my local machine too. Isn't it?

Reply:
Now I get it :) The relative URL is not when working if the user has landed to `http://localhost:3000/wave/docs/examples/all/` instead of `http://localhost:3000/wave/docs/examples/all` and click on a link. But this still works with `useBaseUrl()`

Reply:
Fixed